x-pack/plugins/apm/public/services/rest/ml.ts

```diff
@@ -80,7 +80,7 @@ export async function getHasMLJob({
 }) {
   try {
     await callApi<MLJobApiResponse>({
-      method: 'HEAD',
+      method: 'GET',
```

Comment:
This is a little unfortunate that we have to load the whole body. Did we reach out to the platform team about this?

Reply:
@sqren I never got a chance to, no. I agree, the http service needs to support HEAD/OPTIONS requests.